packages/web-app-simple-editor/src/index.js

```diff
@@ -43,6 +45,13 @@ const appInfo = {
   ]
 }
 
+for (const ext of ['js', 'json', 'xml', 'py', 'php', 'yaml']) {
+  appInfo.extensions.push({
+    extension: ext,
+    ...fileExtensionConfig
+  })
+}
+
```

Comment:
Not a fan of this, maybe we can register editors for mimetypes in the long run? 

As we probably don't have a mechanism for that currently, it's totally out of scope for this PR. Maybe you can open a ticket?

Reply:
Actually, I built registering apps on mime types in https://github.com/owncloud/web/pull/6514 but that one got stale...

Reply:
👍  for that! I would like to be able to open files with no extension.

Reply:
I hope you don't get too overexited with this. Turns out that mime type guessing from the backend isn't always what you'd expect. E.g. ogg video files still get announced with `audio/ogg` as mime type. But I continued the PR and it's close to being merged.